src/view/dataElements/xdmObject/components/nodeEdit.jsx

```diff
@@ -23,28 +23,29 @@ import {
   NUMBER,
   OBJECT
 } from "../constants/schemaType";
-import arrayEdit from "./arrayEdit";
-import booleanEdit from "./booleanEdit";
-import integerEdit from "./integerEdit";
-import numberEdit from "./numberEdit";
-import objectEdit from "./objectEdit";
-import stringEdit from "./stringEdit";
+import ArrayEdit from "./arrayEdit";
+import BooleanEdit from "./booleanEdit";
+import IntegerEdit from "./integerEdit";
+import NumberEdit from "./numberEdit";
+import ObjectEdit from "./objectEdit";
+import StringEdit from "./stringEdit";
```

Comment:
Not really necessary but I noticed they weren't capitalized and it bugged me since they're components.